bigflow/_version.py

```diff
@@ -1,2 +1 @@
-__version__ = '1.5.5dev2'
-
+__version__ = '1.7.0.dev1'
```

Comment:
hmm, doesn't match with the PR title `Release branch 1.6.0`

Reply:
Yeah, but 1.6.0 is already released (I just tagged commit [e6b2c32](https://github.com/allegro/bigflow/pull/360/commits/e6b2c324298a5e0188a1fd1d335a2cecb3de170f)). If I would leave 1.6.0 here it'd try to release it again. What I want to get is to release 1.7.0.dev1 once it's merged to master.